loops-arrays-objects.js

```diff
@@ -74,18 +99,22 @@ const NOVIEmployees = [
 //   { firstName: 'Mark', lastName: 'Rensen', email: 'Mark.Rensen@novi.nl' },
```

Comment:
Het genereren en omzetten van e-mailadressen naar kleine letters is correct gedaan. De code is goed leesbaar.